README.md

```diff
@@ -174,8 +175,16 @@ and the following one to reference this website:
    year = {2019}
 }
 ```
-
-The rest of the publications describes different component of **Crocoddyl**:
+Please also consider citing **[Pinocchio](https://github.com/stack-of-tasks/pinocchio)**, which largely contributes to make **Crocoddyl** an efficient and versatile framework:
```

Comment:
I am not sure if we really need to include the bibtex since this information is already included in Pinocchio repository.
I prefer to keep the real contributions of Crocoddyl, e.g. solvers.
Instead, we should include it in the publication list.

Reply:
I think this is downscaling the role played by Pinocchio, both in terms of performances and versatility. I would really appreciate that people who use Crocoddyl also acknowledge the work made by Pinocchio's developers to support and enhance everything related to robot modelling, collision and differentiation.
In addition, this is also crucial for the current request to get a dedicated software engineer to support Pinocchio and Crocoddyl developments funded by the French institutions.